components/MainNavigation.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import Link from "next/link";
+import styled from "styled-components";
+import Modal from "./modal/Modal";
+
+class MainNavigation extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isModalOpen: false
+        }
+    }
+
+    onOpenModal = () => {
+        this.setState({ isModalOpen: true });
+    }
+
+    onCloseModal = () => {
+        this.setState({ isModalOpen: false });
+    }
+
+    render() {
+        return (
+            <Container>
+                <Link href="/">
+                    <a>Velog</a>
+                </Link>
+                <main>
+                    <Button onClick={this.onOpenModal}>로그인</Button>
+                    <Modal isOpen={this.state.isModalOpen} isClose={this.onCloseModal} />
+                </main>
+            </Container>
+        )
+    }
+}
+
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  left: 0;
+  top: 0;
+  margin: 20px;
+  justify-content: space-between;
+`;
+
+const Button = styled.span`
+  top: 16px;
+  right: 904px;
+  left: 830px;
+  bottom: 48px;
```

Comment:
```suggestion
  top: 16px;
  right: 904px;
  left: 830px;
  bottom: 48px;
```
포지션 영역이 없는데 위 속성들이 잘 들어가고 있나요 ?

Reply:
오 감사합니다. 캐치하지 못했네요.. 수정했습니다!